src/mon/Monitor.cc

```diff
@@ -5584,14 +5583,19 @@ int Monitor::scrub_start()
   dout(10) << __func__ << dendl;
   ceph_assert(is_leader());
 
-  if (!scrub_result.empty()) {
-    clog->info() << "scrub already in progress";
-    return -EBUSY;
+  if (scrub_con.load() != nullptr) {
+    std::shared_ptr<ScrubContext> local_con = scrub_con.load();
+    if (local_con) {
```

Comment:
`local_con` sounds like _local connection_ but I believe the idea was about `context`. Perhaps `local_context` or even `local_ctx` would do.

Reply:
done